common/views/components/StackingTable/StackingTable.tsx

```diff
@@ -89,7 +89,7 @@ const StyledTh = styled(Space).attrs<ThProps>(props => ({
 type TdProps = {
   plain?: boolean;
   maxWidth?: number;
-  content?: string | ReactNode;
+  cellContent?: ReactNode;
```

Comment:
This was being flagged by Typescript. `content` is a React property and it wasn't matching its type to our custom one, so I changed its name.

Reply:
Also, `string` is a `ReactNode`, so simplified it